pkg/kubectl/run.go

```diff
@@ -121,94 +122,6 @@ func (DeploymentV1Beta1) Generate(genericParams map[string]interface{}) (runtime
 	return &deployment, nil
 }
 
-type DeploymentAppsV1Beta1 struct{}
```

Comment:
Removing `DeploymentAppsV1Beta1` because it's not being used anywhere, and we have plans to deprecate `kubectl run`.

Reply:
how do you know it's not being used anywhere? it's usable by specifying the generator name to the run command, right?

Reply:
Please look at https://github.com/kubernetes/kubernetes/blob/02a80b6c5e6efd427f834f11ca97e094d4c7fa0a/pkg/kubectl/cmd/util/factory_client_access.go#L554, there's no `DeploymentAppV1Beta1`

Reply:
My bad, will revert this.

Reply:
> there's no DeploymentAppV1Beta1

that's because you removed it in this commit :)

https://github.com/kubernetes/kubernetes/pull/61288/files?w=1#diff-61ef000e48b9358fadec0877c58832fbL557

it is used in released versions:

https://github.com/kubernetes/kubernetes/blob/fc32d2f3698e36b93322a3465f63a14e9f0eaead/pkg/kubectl/cmd/util/factory_client_access.go#L517-L526

Reply:
Oops! :)

I probably got confused because we don't fall back in case there is no `apps/v1beta1`
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/util/factory_client_access.go#L559

Will fix this.

Reply:
Is that because if the user specified the generator then we don't fall back?

Reply:
> Is that because if the user specifies the generator then we don't fall back?

something like that. they explicitly asked for a specific output, and we probably should not change what they asked for.

Reply:
We do the same for deployments - https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/create_deployment.go#L115
Should I add fall back for `DeploymentAppV1Beta1` for consistency?